packages/backend-core/src/utils/tests/utils.spec.ts

```diff
@@ -106,4 +108,79 @@ describe("utils", () => {
       expect(actual).toBe(undefined)
     })
   })
+
+  describe("isServingBuilder", () => {
+    let ctx: BBContext
+
+    const expectResult = (result: boolean) =>
+      expect(utils.isServingBuilder(ctx)).toBe(result)
+
+    beforeEach(() => {
+      ctx = structures.koa.newContext()
+    })
+
+    it("returns true if current path is in builder", async () => {
+      ctx.path = "/builder/app/app_"
+      expectResult(true)
+    })
+
+    it("returns false if current path is not in builder", async () => {
```

Comment:
might be more clear:
```suggestion
    it("returns false if current path doesn't have '/' suffix", async () => {
```